### PR TITLE
[nibeheatpump] Removed dependency on 'org.apache.commons.io.IOUtils'

### PR DESCRIPTION
**Pull request by lolodomo**
Relative to #7722

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.nibeheatpump/src/main/java/org/openhab/binding/nibeheatpump/internal/connection/SerialConnector.java [hunk 1]
```diff
@@ -163,7 +172,6 @@ public void sendDatagram(NibeHeatPumpMessage msg) {
         public void interrupt() {
             interrupted = true;
             super.interrupt();
-            IOUtils.closeQuietly(in);
```
Comment (J-N-K) [inline bundles/org.openhab.binding.nibeheatpump/src/main/java/org/openhab/binding/nibeheatpump/internal/connection/SerialConnector.java:166]: Why did you remove that?

Comment (lolodomo) [reply bundles/org.openhab.binding.nibeheatpump/src/main/java/org/openhab/binding/nibeheatpump/internal/connection/SerialConnector.java:166]: In fact, the input stream is managed (opened and closed) by the thing handler, not by the reader thread started by the thing handler. When the conection is closed by thing handler, it first interrupts the reader thread (suing the input stream) and then closes the input stream and the serial port. This is what you see in the disconnect() method.

All bindings that I know with a reader thread are doing that like that.

Note that I applied the same change in the rme and oceanic bindings.
